technical-reports/format/composite-types.md

```diff
@@ -1,57 +1,72 @@
 # Composite types
 
-<div class="issue" data-number="54" title="Should composites be part of the MVP specification?">
+The types defined in the previous chapters such as color and dimension all have singular values. For example, the value of a color token is _one_ color. However, there are other aspects of UI designs that are a combination of multiple values. For instance, a shadow style is a combination of a color, X & Y offsets, a blur radius and a spread radius.
 
-If so, which composites should be included initially?
+Every shadow style has the exact same parts (color, X & Y offsets, etc.), but their respective values will differ. Furthermore, each part's value (which is also known as a "sub-value") is always of the same type. A shadow's color must always be a [color](#color) value, its X offset must always be a [dimension](#dimension) value, and so on. Shadow styles are therefore combinations of values _that follow a pre-defined structure_. In other words, shadow styles are themselves a type. We call types like this **composite types**.
```

Comment:
Minor: should we not use "we" in the spec? "We call types like this..." v. "Types like this are called..."

Reply:
I suggest we shouldn't use "we" in the spec.

Reply:
Agreed. I'll fix that.